dev-libs/libaddressinput/libaddressinput-0~git.recipe

```diff
@@ -0,0 +1,88 @@
+SUMMARY="Google’s postal address library, powering Android and Chromium"
```

Comment:
`SUMMARY="Google’s postal address library that powers Android and Chromium"`
This would make the `SUMMARY` sound more grammatically correct :)